### ISSUE TITLE
Adding DragDropContext just breaks everything

### ISSUE THREAD

**Issue by jeron-diovis** [gaearon/react-dnd#258]
Just added DragDropContext to my topmost component:
```jsx
import HTML5Backend from 'react-dnd/modules/backends/HTML5';
import { DragDropContext } from 'react-dnd';

@DragDropContext(HTML5Backend)
export default class App extends React.Component {
   ...
}
```

On rendering got this:
```
Uncaught TypeError: Cannot read property 'hasOwnProperty' of undefined 
ReactElementValidator.js:157:27 
```

Discouraging.

Feeling myself dumb, but am I doing something wrong?

**Comment by jeron-diovis** [comment]
Actually, even if have a single component with one lonely <div> inside, the same happens.

**Comment by gaearon** [comment]
Please check if you have two Reacts in the compiled bundle.
It's a common problem with NPM. (Hopefully to be solved in NPM 3).

See here for more info: https://medium.com/@dan_abramov/two-weird-tricks-that-fix-react-7cf9bbdef375

It's definitely not a React DnD issue.

**Comment by jeron-diovis** [comment]
Indeed. 
`"peerDependencies": {  "react": ">=0.13.0 <0.15.0 || 0.14.0-beta1" }` was resolved to 0.14.0-beta1.
It seems to be my package manager problem (jspm in case), because with just `npm install react-dnd` I have 0.13.3 installed.
Fixed it with shrinkwrap. Thank you for advice.

**Comment by gaearon** [comment]
No problem, glad it helped!
